spec/state_machines/verification_attempt_state_machine_spec.rb

```diff
@@ -6,21 +6,28 @@
     let(:user) { create :admin_user }
 
     context "to approved" do
+      before do
+        verification_attempt.client.touch(:identity_verification_denied_at)
```

Comment:
how does .touch work?

Reply:
.touch updates this column in the db to be the current timestamp.